app/helpers/network_port_helper/textual_summary.rb

```diff
@@ -45,8 +53,9 @@ def textual_ems_network
   def textual_instance
     label    = ui_lookup(:table => "vm_cloud")
     instance = @record.device
-    h        = {:label => label, :image => "vm"}
+    h        = nil
```

Comment:
What's the impact of returning `nil` here?  Does it just not show the `instance` relationship box?